qa/integration-tests/src/test/java/io/camunda/it/client/UserTaskQueryTest.java

```diff
@@ -176,15 +176,16 @@ public void shouldRetrieveTaskByTenantId() {
   }
 
   @Test
-  public void shouldRetrieveTaskByPriority() {
-    final var resultDefaultTenant =
-        camundaClient
-            .newUserTaskQuery()
-            .filter(f -> f.priority(new PriorityValueFilter().eq(30)))
-            .send()
-            .join();
-    assertThat(resultDefaultTenant.items().size()).isEqualTo(1);
-    assertThat(resultDefaultTenant.items().getFirst().getPriority()).isEqualTo(30);
+  public void retrievedTasksShouldIncludePriority() {
+    final var resultDefaultPriority =
+        camundaClient.newUserTaskQuery().filter(f -> f.bpmnProcessId("process-2")).send().join();
+    assertThat(resultDefaultPriority.items().size()).isEqualTo(1);
+    assertEquals(50, resultDefaultPriority.items().getFirst().getPriority());
```

Comment:
🔧 Let's not mix AssertJ and JUnit assertions, if possible. There is another occasion a couple of lines later.
```suggestion
    assertThat(resultDefaultPriority.items()).hasSize(1);
    assertThat(resultDefaultPriority.items().getFirst().getPriority()).isEqualTo(50);
```